From: Eduardo Pereira Habkost <ehabkost@redhat.com>
Date: Tue, 16 Sep 2008 15:17:48 -0300
Subject: [xen] virtio: include headers in kernel-headers package
Message-id: 20080916181747.GK12539@blackpad
O-Subject: [RHEL5.3 PATCH] virtio: Include virtio headers in kernel-headers package
Bugzilla: 446214
RH-Acked-by: Mark McLoughlin <markmc@redhat.com>
RH-Acked-by: Bill Burns <bburns@redhat.com>
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Chris Wright <chrisw@redhat.com>

From: Chris Lalancette <clalance@redhat.com>

https://bugzilla.redhat.com/446214

The virtio code exports several headers to userspace, but
the backport patch neglected to include them the list of
headers to be included in kernel-headers.

Tested by building in brew and confirming the headers
are now included in the kernel-headers package.

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>

diff --git a/include/linux/Kbuild b/include/linux/Kbuild
index e910a49..e7403e6 100644
--- a/include/linux/Kbuild
+++ b/include/linux/Kbuild
@@ -346,6 +346,11 @@ unifdef-y += user.h
 unifdef-y += utsname.h
 unifdef-y += videodev2.h
 unifdef-y += videodev.h
+unifdef-y += virtio_config.h
+unifdef-y += virtio_blk.h
+unifdef-y += virtio_net.h
+unifdef-y += virtio_pci.h
+unifdef-y += virtio_ring.h
 unifdef-y += wait.h
 unifdef-y += wanrouter.h
 unifdef-y += watchdog.h
